Lab4/ImageLab/OpenCVBridgeFace.mm

```diff
@@ -0,0 +1,39 @@
+//
+//  OpenCVBridgeSub.m
+//  ImageLab
+//
+//  Created by Eric Larson on 10/4/16.
+//  Copyright © 2016 Eric Larson. All rights reserved.
+//
+
+#import "OpenCVBridgeFace.h"
+
+#import "AVFoundation/AVFoundation.h"
+
+
+using namespace cv;
+
+@interface OpenCVBridgeFace()
+@property (nonatomic) cv::Mat image;
+@end
+
+@implementation OpenCVBridgeFace
+@dynamic image;
+//@dynamic just tells the compiler that the getter and setter methods are implemented not by the class itself but somewhere else (like the superclass or will be provided at runtime).
+
+-(void)processImage{
+    
+    cv::Mat frame_gray,image_copy;
+    char text[50];
+    Scalar avgPixelIntensity;
+    cv::Mat image = self.image;
+    
+    cvtColor(image, image_copy, CV_BGRA2BGR); // get rid of alpha for processing
```

Comment:
Should we make the mask CV_BGRA2RGB so we are ready for processing colors correctly? (based off the ICA)

Reply:
Nice catch. Fixed.